### DISCUSSION TITLE
Rubberbanding issues

### DISCUSSION THREAD

**Discussion by mwessman** [wolveix/satisfactory-server#295] [Q&A] [answered]
So I'm currently hosting my docker container on GCP on a e2-standard-4 instance (4vCPU 16GB memory) and I'm suffering extreme rubberbanding already after a few hours. Even when I'm solo on the server and exploring, the rubberbanding of enemies is crazy.

I increased the memory limit of the docker container file, but it's still lagging. When monitoring the CPU and Memory usage of the container, it hovers around 20-40% usage for both.
The VM region is also the same country I am in, so I doubt the issue has to do with network lag.

Has anyone faced similar issues and found a solution for it?

**Comment by wolveix** [comment, marked as answer]
Hey!

Satisfactory requires a CPU with **strong** single-core performance. I strongly suspect that that is your issue. Additionally, check the network setting in the server manager's advanced settings :)

**Comment by mwessman** [comment]
Thanks for the help. The network settings didn't really help that much, but changing the instance from a "standard" instance to a slightly more expensive high performance VM made all the difference. It's still lagging every now and then, but it's not nearly as bad as it was before.

**Reply by wolveix** [reply]
Yes, it's due to the requirement of strong single-core performance. Glad it's better now!
